src/components/TextEditor.tsx

```diff
@@ -113,18 +104,6 @@ export const TextEditor = ({
   // const onChange = React.useCallback((value: string, viewUpdate: ViewUpdate) => {
   const onChange = (newCode: string) => {
     kclManager.setCodeAndExecute(newCode)
-    if (isTauri() && pathParams.id) {
-      // Save the file to disk
-      // Note that PROJECT_ENTRYPOINT is hardcoded until we support multiple files
-      writeTextFile(pathParams.id, newCode).catch((err) => {
-        // TODO: add Sentry per GH issue #254 (https://github.com/KittyCAD/modeling-app/issues/254)
-        console.error('error saving file', err)
-        toast.error('Error saving file, please check file permissions')
-      })
-    }
-    if (editorView) {
-      editorView?.dispatch({ effects: addLineHighlight.of([0, 0]) })
-    }
```

Comment:
`kclManager.setCodeAndExecute(newCode)` handles saving the file too now, hence why all this could be deleted.